.github/pull_request_template.txt

```diff
@@ -0,0 +1,18 @@
+## Problem
+- Describe the problem this PR solves here.
+- If you're fixing an issue, say "Fixes #X".
```

Comment:
problem section is not always applicable, and can often be redundant

Reply:
When would it not be applicable? 
Every PR has a problem it's trying to solves, as that's the whole point of the PR?

Reply:
Though to be fair, `problem` may not be the best name for this?
Alternatives: 
`issue` 
`goal`

Reply:
I don't like issue due to the overloading.

Goal or objective seem nice; although I think it might make sense to collapse this with the solution.

Reply:
goal or objective feel more positive at least 👍